src/Tests/InstrEngineTests/InstrEngineTests/EmbeddedResources/ByRefLikeInvalidCSharp.il

```diff
@@ -0,0 +1,631 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+.assembly extern System.Console { .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A ) }
+.assembly extern System.Runtime { .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A ) }
+
+.assembly InvalidCSharp { }
```

Comment:
Was the name InvalidCSharp meant to suggest this can't be expressed in csharp or is actually invalid? I found this confusing.

Reply:
I inherited the name from Aaron's code. Some of the stuff cannot be expressed in CSharp. Some of it is actually invalid. Some of the tests that are written are to ensure that the jitter throws an exception when it attempts to execute. We could potentially remove those, since we aren't concerned with invalid IL for CLRIE